run_clang_analyze.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I'm not a fan putting auxiliary scripts in the root, you better off to place them in a scripts folder and add a small note what it does.